common/modal.js

```diff
@@ -309,6 +309,7 @@
         function SearchPopupController(ConfigUtils, DataUtils, params, Session, logService, modalBox, recordsetDisplayModes, $rootScope, $timeout, $uibModalInstance) {
         var vm = this;
 
+        vm.rowsOnLoad = params.selectedRows.length > 0;
```

Comment:
Can you please add a comment explaining why this variable exists? At least I'm sure that I won't remember it by just looking at its name.